app/assets/javascripts/components/tabs.js

```diff
@@ -0,0 +1,106 @@
+/*
+    We used the thorough analysis and scripting of Jason Kiss for these ARIA tabs.
+    See Accessible ARIA Tabs: http://www.accessibleculture.org/articles/2010/08/aria-tabs/
+    for his analysis, examples and conclusions. Thanks, Jason!
+*/
+$(function() {
+    'use strict';
+    var tabs = $("#tabs");
+
+    // For each individual tab DIV, set class and aria role attributes, and hide it
+    $(tabs).find("> div").attr({
+        "role": "tabpanel",
+        "aria-hidden": "true"
+    }).addClass("tabPanel").hide();
+
+    // Get the list of tab links
+    var tabsList = tabs.find("ul:first").attr({
+        "class": "tabsList",
+        "role": "tablist"
+    });
+
+    // For each item in the tabs list...
+    $(tabsList).find("li > a").each(
+        function(a){
+            var tab = $(this);
+
+            // Create a unique id using the tab link's href
+            var tabId = "tab-" + tab.attr("href").slice(1);
+
+            // Assign tab id, aria and tabindex attributes to the tab control, but do not remove the href
+            tab.attr({
+                "id": tabId,
+                "role": "tab",
+                "aria-selected": "false",
+                "tabindex": "-1"
+            }).parent().attr("role", "presentation");
+
+            // Assign aria attribute to the relevant tab panel
+            $(tabs).find(".tabPanel").eq(a).attr("aria-labelledby", tabId);
+
+            // Set the click event for each tab link
+            tab.click(
+                function(e){
+                    // Prevent default click event
+                    e.preventDefault();
+
+                    // Change state of previously selected tabList item
+                    $(tabsList).find("> li.current").removeClass("current").find("> a").attr({
```

Comment:
The string `'current'` is used in a handful of places. It may be wise to stash that class name in a variable:

```js
var currentClassName = 'current';
```

…and use it like:

```js
$tabsList.find('> li.current').removeClass(currentClassName);
```

`currentClassName` can then be munged by Uglifer to something like… `f`. As it stands, the string `'current'` can't be shortened by Uglifier.